MovieSaver/Services/IOService.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  IOService.swift
+//  MovieSaver
+//
+//  Created by Yushkevich Ilya on 10.04.22.
+//
+
+import Foundation
+
+protocol IOService {
```

Comment:
Bad name for protocol. It's better to rename it to `MoviesRepositoryProtocol` or just `MoviesRepository`